build.rs

```diff
@@ -44,7 +49,7 @@ fn generate_pb(path: std::path::PathBuf) {
     let _ = protos_cmd
```

Comment:
prettu sure you can get rid of the `let _ =` too

Reply:
fixed

Reply:
Shoot, i'll get rid of the comment too.